### PR TITLE
Reduce amount of scroll bars.

### PR DESCRIPTION
**Pull request by kbradl16**
Make the app feel more like 1 part instead of 2 parts.

Reduce amount of scrolling, reduce annoyance factor. :) 

p.s. Love the app!!!

scrollbar galore
Lots of scrolling

2 scroll bars
After My edits

### CODE REVIEW

**Comment by maxcnunes** [comment]
Awesome contributing. Right now I am on vocations. So I will take a look on this when I get back to work. Mean while could you check if the sidebar still scrolling separately from the right? It was a feature some people asked me (https://github.com/sqlectron/sqlectron-gui/issues/65).

Thanks!

**Comment by kbradl16** [comment]
Ah, that makes sense... 
I've come up with a better solution to keep the 2 separate scroll bars and still reduce the overall amount.

**Comment by kbradl16** [comment]
How it looks after edits
newlook

File: src/renderer/containers/query-browser.jsx [hunk 1]
```diff
@@ -22,10 +22,11 @@ require('../components/react-tabs.scss');
 
 
 const STYLES = {
-  wrapper: { paddingTop: '50px' },
-  container: { display: 'flex', padding: '10px 10px 50px 10px' },
-  sidebar: { overflowY: 'auto', height: '100vh' },
-  content: { flex: 1, overflow: 'auto', height: '100vh' },
+  wrapper: {  },
+  container: { display: 'flex', height: '100vh', box-sizing: 'border-box', padding: '50px 10px 40px 10px'  },
```
Comment (maxcnunes) [inline src/renderer/containers/query-browser.jsx]: `box-sizing` is not a valid react style attribute https://facebook.github.io/react/tips/inline-styles.html

File: src/renderer/containers/query-browser.jsx [hunk 2]
```diff
@@ -22,10 +22,11 @@ require('../components/react-tabs.scss');
...
+  sidebar: { overflowY: 'auto' },
+  resizeable: { max-width: '100%', padding-right: '5px' },
```
Comment (maxcnunes) [inline src/renderer/containers/query-browser.jsx]: `max-widht` and `paddign-right` are not valid react style attributes

File: src/renderer/containers/query-browser.jsx [hunk 3]
```diff
@@ -22,10 +22,11 @@ require('../components/react-tabs.scss');
...
+  content: { flex: 1, overflow: 'auto', padding-left: '5px' },
```
Comment (maxcnunes) [inline src/renderer/containers/query-browser.jsx]: `padding-left` is not a valid react style attribute

File: src/renderer/containers/query-browser.jsx [hunk 4]
```diff
@@ -269,7 +270,7 @@ class QueryBrowserContainer extends Component {
             <ResizableBox className="react-resizable react-resizable-ew-resize"
               width={235}
               minConstraints={[235, 300]}
-              maxConstraints={[750, 10000]}>
+              maxConstraints={[750, 10000]} style={STYLES.resizeable}>
```
Comment (maxcnunes) [inline src/renderer/containers/query-browser.jsx]: `ResizableBox` does not support `style` property. Actually it is crashing the `ResizableBox`.

**Comment by maxcnunes** [comment]
Even with some problems in your pull request I will accept it and apply the fixing myself since I already know how to fix them.

**Comment by maxcnunes** [comment]
Done. Let me know if it still working as you were expecting.

**Comment by kbradl16** [comment]
Ooops, sorry bout that.  I was making my best guesses... I just assumed it was like valid css styles.

**Pull request merged**